x-pack/plugins/rule_registry/server/event_log/elasticsearch/index_writer.ts

```diff
@@ -85,10 +93,12 @@ export class IndexWriter {
         error.stack += '\n' + util.inspect(response.body.items, { depth: null });
         this.logger.error(error);
       }
+      return response.body;
     } catch (e) {
       this.logger.error(
         `error writing bulk events: "${e.message}"; docs: ${JSON.stringify(bulkBody)}`
       );
+      return undefined;
```

Comment:
We'll probably need to return the error here or rethrow it. Just logging errors usually don't suffice.